src/plugins/quickdump/benchmarks/v2/results.get.200k.json

```diff
@@ -0,0 +1,40 @@
+{
```

Comment:
Please add this data to the rawdata.libelektra.org repo.

Reply:
@markus2330 It seems I don't have write access to that repo

Reply:
I'm only waiting on the access to the repo, the failing test is because the link will only work once I can push to the rawdata repo...

Reply:
You can also open a pull request at the raw data repository. At least that is what I do, if I want to add benchmarks scripts and results.

Reply:
I opened ElektraInitiative/rawdata#6 for the results files, the input files are https://github.com/kodebach/eqd-bench since git LFS didn't work with the PR.

Reply:
Thank you, I now added ElektraDevelopers with write permissions to the repo. Tell me if pushing the LFS files did not work.